packages/chain-adapters/src/ChainAdapterManager.ts

```diff
@@ -38,7 +38,23 @@ export class ChainAdapterManager {
             const ws = new unchained.bitcoin.ws.Client(wsUrl)
             return this.addChain(
               type,
-              () => new bitcoin.ChainAdapter({ providers: { http, ws }, coinName: 'Bitcoin' })
+              () => new bitcoin.ChainAdapter({ providers: { http, ws }, coinName: 'Cosmos' })
```

Comment:
Still seeing this issue
```suggestion
              () => new bitcoin.ChainAdapter({ providers: { http, ws }, coinName: 'Bitcoin' })
```